loopd/run.go

```diff
@@ -82,7 +82,7 @@ func newListenerCfg(config *Config, rpcCfg RPCConfig) *listenerCfg {
 		getLnd: func(network lndclient.Network, cfg *lndConfig) (
 			*lndclient.GrpcLndServices, error) {
 
-			syncCtx, cancel := context.WithCancel(
+			clCtx, cancel := context.WithCancel(
```

Comment:
very nit, why not `callCtx` ? :)

Reply:
Originally called this client ctx in lndclient, but decided there were too many things called client :) updated to `callerCtx`